src/handles.cpp

```diff
@@ -60,7 +60,7 @@ NTSTATUS _CreateHandle(
 #pragma warning(suppress : 26492) // const_cast is prohibited, but we can't avoid it for filling UNICODE_STRING.
     Name.Buffer = (wchar_t*)DeviceName;
     Name.Length = (USHORT)((wcslen(DeviceName) * sizeof(wchar_t)));
-    Name.MaximumLength = Name.Length + sizeof(wchar_t);
+    Name.MaximumLength = Name.Length;
```

Comment:
I think it was right as is, since `DeviceName` is a null terminated string and `Name.Length` is set to `wcslen` which doesn't count null terminator.

Reply:
Ugh.. My bad! I misread the operator, thought it was _multiplying_ by the sizeof again and it was kind of a copy pasta situation. Will revert that right now.

Reply:
No worries, since you made revert a separate commit (vs patching up previous commit), I'll squash merge changes, so all of them will be a single commit.